packages/template/typescript/src/TypeScriptTemplate.ts

```diff
@@ -0,0 +1,51 @@
+import { ForgeTemplate } from '@electron-forge/shared-types';
+import { asyncOra } from '@electron-forge/async-ora';
+
+import fs from 'fs-extra';
+import path from 'path';
+
+const copyTemplateFile = async (destDir: string, basename: string) => {
+  const templateDir = path.resolve(__dirname, '..', 'tmpl');
+  await fs.copy(path.join(templateDir, basename), path.resolve(destDir, basename));
+};
```

Comment:
I wonder if there needs to be a `TemplateBase` in the same way we have base classes for makers and publishers. Then a variant of this function could become a method in that class.

Reply:
good q - i was going to make a combination ts-webpack template after this so i could do a refactor there potentially?

Reply:
Sounds good to me.

Reply:
I've done this in https://github.com/electron-userland/electron-forge/pull/1351 !